### PR TITLE
Add donna hay

### PR DESCRIPTION
**Pull request by Mooree003**
I've added functionality for all the fields i can find on the site, and deleted all irrelevant ones. The ratings system on the site is a dummy version and you can't actually interact with it so i didn't include it in the class

### CODE REVIEW

File: recipe_scrapers/donnahay.py [hunk 1]
```diff
@@ -8,23 +8,23 @@ class DonnaHay(AbstractScraper):
...
+        return (
+            self.soup.find("h1", class_="text-center recipe-title__mobile")
+            .getText()
+            .upper()
+        )
 
     def yields(self):
-        return self.schema.yields()
+        div = self.soup.find("div", class_="col-sm-6 method")
+        return div.find("b").getText()
```
Comment (a1831319) [inline recipe_scrapers/donnahay.py]: Some of the older recipes such as https://www.donnahay.com.au/recipes/lunch/asian-style-prawn-and-rice-omelette don't have the serves in a separate \<b\> tag. Might need to get this detail another way.

File: recipe_scrapers/donnahay.py [hunk 2]
```diff
@@ -8,23 +8,23 @@ class DonnaHay(AbstractScraper):
     def host(cls):
         return "donnahay.com.au"
 
-    def author(self):
-        return self.schema.author()
```
Comment (a1831319) [inline recipe_scrapers/donnahay.py:12]: Could probably keep this and have it return the site name based on the instructions from the scraper functions guide (https://github.com/hhursev/recipe-scrapers/blob/main/docs/in-depth-guide-scraper-functions.md)

**Review by a1831319 — COMMENTED** [review]
Looks generally pretty good, just a couple comments.

**Review by a1831319 — APPROVED** [review]
Looks good with the changes, very nice.

**Review by heathrampazis — APPROVED** [review]
Looks good.

**Pull request merged**